src/psyclone/domain/lfric/kernel/field_arg.py

```diff
@@ -55,6 +55,10 @@ class FieldArg(CommonArg):
 
     '''
     form = "GH_FIELD"
```

Comment:
Please add comments documenting what each of these class variables holds.

Reply:
Done.